releasenotes/notes/refactor-basicaer-to-basicprovider-e27aff9c8e81d26e.yaml

```
@@ -0,0 +1,91 @@
+---
+features:
+  - |
+    A new :mod:`qiskit.providers.basic_provider` module has been introduced
+    to replace ``qiskit.providers.basicaer``. This module contains provider
+    tools that mirror those of the ``BasicAer`` provider and offers a single,
+    non-efficient, statevector based simulator: :class:`.BasicSimulator`.
+    This simulator based on the :class:`.BackendV2` interface and is
+    exclusively intended for testing and simple prototyping, for more
+    advanced simulation capabilities, please refer to the ``qiskit-aer``
+    package.
+
+  - |
+    The :class:`.Target` interface and transpiler pipeline now support
+    target definitions with ``num_qubits=None``. This is to support the
+    creation of target-based simulators with a flexible number of qubits.
```

Uppercase `Target` (w/o the link) would also be good I think
```suggestion
    creation of :class:`.Target`-based simulators with a flexible number of qubits.
```